packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -395,6 +394,9 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
         status_ = claimData[0].counteredBy == address(0) ? GameStatus.DEFENDER_WINS : GameStatus.CHALLENGER_WINS;
         resolvedAt = Timestamp.wrap(uint64(block.timestamp));
 
+        // Try to update the anchor state, but do not revert if it fails.
+        try ANCHOR_STATE_REGISTRY.tryUpdateAnchorState() { } catch { }
```

Comment:
hrmm. I'd prefer here if we removed the `require` for `game.l2BlockNumber() > anchors[gameType].l2BlockNumber` + retain the other requires. This should bubble up the revert on the other two checks.